examples/app/headless_defaults.rs

```diff
@@ -1,3 +1,5 @@
+//! An application that runs with default plugins, but without an actual renderer
```

Comment:
```suggestion
//! An application that runs with default plugins, but without an actual renderer
//! This can be very useful for integration tests or CI.
```